packages/runtime/package.json

```diff
@@ -52,10 +52,14 @@
       "import": "./dist/esm/next/index.js",
       "types": "./dist/types/next/index.d.ts"
     },
+    "./next/server": {
+      "require": "./dist/cjs/next/server.js",
+      "module": "./dist/esm/next/server.js",
+      "types": "./dist/types/next/server.d.ts"
+    },
     "./next/plugin": {
-      "require": "./node_modules/@makeswift/next-plugin/index.js",
-      "import": "./node_modules/@makeswift/next-plugin/index.js",
```

Comment:
Are we not exposing a ES module anymore?

Reply:
Not for the Next.js plugin. It's a CommonJS module right now and if we want to expose an ES module we'll need to do some work so that `@makewift/next-plugin` exports an ES module as well.